packages/system/src/sprinkles/sprinkles.css.ts

```diff
@@ -201,6 +201,14 @@ const layout = defineProperties({
 				display: "flex",
 				flexDirection: "row",
 			},
+
```

Comment:
I would suggest not touching this file at all. 
It would be better to only add styles to the `contributor-banner.css.ts` file since that is the component you are working with.